tests/kernels/test_attention.py

```diff
@@ -96,16 +97,17 @@ def ref_single_query_cached_kv_attention(
         output[i].copy_(out, non_blocking=True)
 
 
+@pytest.mark.parametrize("use_v2", [True, False])
```

Comment:
```suggestion
@pytest.mark.parametrize("version", ["v1", "v2"])
```
And the code below can be changed accordingly

Reply:
Fixed!